app/assets/javascripts/search.js

```diff
@@ -1,18 +1,19 @@
 $(document).ready(function() {
 
-  var searchUI = $(".m-search-ui");
-  var advOptsFieldset = $("fieldset.adv");
-  var advOptsControlsFieldset = $("fieldset.controls");
-  var searchTerms = $(".m-search-ui .search-terms");
-  var searchButton = $("#search-button");
-  var searchMagGlass = $(".m-search-ui .input-group-addon.magnifier");
-  var advOptsButton = $("#adv-options");
-  var advOptsToggler = $("a.adv-options");
-  var advOptsCloser = $(".adv-controls .closer");
-  var allSearchButtons = $(".m-search-ui button.search");
-  var searchForm = $("form.search");
-  var countEl = $(".results-count-preview .count");
-  var advOptsResetter = $("a.resetter");
+  var searchUI = $(".m-search-ui"),
+  advOptsFieldset = $("fieldset.adv"),
+  advOptsControlsFieldset = $("fieldset.controls"),
+  searchTerms = $(".m-search-ui .search-terms"),
+  searchButton = $("#search-button"),
+  searchMagGlass = $(".m-search-ui .input-group-addon.magnifier"),
+  advOptsButton = $("#adv-options"),
+  advOptsToggler = $("a.adv-options"),
+  advOptsCloser = $(".adv-controls .closer"),
+  allSearchButtons = $(".m-search-ui button.search"),
+  searchForm = $("form.search"),
+  countEl = $(".results-count-preview .count"),
+  advOptsResetter = $("a.resetter"),
+  formHasFocus = false;
```

Comment:
@dogweather There is a comma at the end of each assignment, so the first `var` is being applied to everything that follows.